### PR TITLE
Update build instructions for Linux.

### PR DESCRIPTION
**Pull request by vkucera**

### CODE REVIEW

File: building/build.md [hunk 1]
```diff
@@ -209,7 +209,6 @@ _⚠️  ROOT 5 does not work on macOS. Use a ROOT 6 version compatible with ROO
 
 ```bash
 aliBuild build AliPhysics --defaults user-root6  # without GEANT 3, GEANT 4, DPMJET
```
Comment (mpuccio) [inline building/build.md:211]: Can you please add user-next-root6 here?

**Review by mpuccio — CHANGES_REQUESTED** [review]

File: building/build.md (hunk 1 above)
Comment (vkucera) [reply building/build.md:211]: `user-next-root6` is recommended as the default build option.
This is the **Build AliPhysics based on an old ROOT 6 version:** section.
Can you be more specific?

Comment (mpuccio) [reply building/build.md:211]: yes, @ktf shall we remove the part on the old ROOT from the documentation? I would be very much in favour.

Comment (ktf) [reply building/build.md:211]: You mean ROOT5? Given this is the analysis tutorial I guess we can.

Comment (mpuccio) [reply building/build.md:211]: This is even worse, it's about old ROOT 6 versions. But OK, I think we can remove this, @vkucera can you do it? Just remove the full section `Build AliPhysics based on an old ROOT 6 version`

**Review by mpuccio — APPROVED** [review]

**Pull request merged**